bigbone/src/main/kotlin/social/bigbone/api/entity/Suggestion.kt

```diff
@@ -0,0 +1,46 @@
+package social.bigbone.api.entity
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ * Represents a suggested account to follow and an associated reason for the suggestion.
+ * @see <a href="https://docs.joinmastodon.org/entities/Suggestion/">Mastodon API Suggestion</a>
+ */
+
+@Serializable
+data class Suggestion(
+
+    /**
+     * The reason this account is being suggested.
+     */
+    @SerialName("source")
+    val source: String = SourceSuggestion.STAFF.value,
```

Comment:
I’d suggest to instead return the `SourceSuggestion` enum type here instead.
```suggestion
    val source: SourceSuggestion = SourceSuggestion.STAFF,
```